screener-runner/README.md

```diff
@@ -0,0 +1,10 @@
+# Screener Runner
+This repository contains examples of how to use [Screener Runner](https://github.com/screener-io/screener-runner)
```

Comment:
Can you also add more information on how to set up an account for Screener (maybe just the wiki link),because we mention the sauce user account details in the main readme.

Then it's good to go